src/ds/ds5/ds5-color.cpp

```diff
@@ -199,17 +169,27 @@ namespace librealsense
                     _thermal_monitor->add_observer([&](float) {
                     _color_calib_table_raw.reset(); });
             }
+        }
 
-            if (_pid != ds::RS457_PID) //not mipi device
```

Comment:
Hard to follow, where did this lines go?

Reply:
In line 191, in method register_metadata